src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1079,7 +1079,10 @@ public virtual SignOutResult SignOut(AuthenticationProperties properties, params
                 throw new ArgumentNullException(nameof(prefix));
             }
 
-            return TryUpdateModelAsync(model, prefix, new CompositeValueProvider(ControllerContext.ValueProviders));
+            var valueProvider = await CompositeValueProvider.CreateAsync(
+                ControllerContext,
+                ControllerContext.ValueProviderFactories);
```

Comment:
Suggest reducing this to `await CompositeValueProvider.CreateAsync(ControllerContext)`. No need for the additional parameter at least here.